cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/common/DataprocUtils.java

```diff
@@ -63,7 +63,9 @@ public final class DataprocUtils {
   public static final String CDAP_GCS_ROOT = "cdap-job";
   public static final String CDAP_CACHED_ARTIFACTS = "cached-artifacts";
   public static final String WORKER_CPU_PREFIX = "Up to";
-  public static final String DISABLE_GCS_CACHING = "disableGCSCaching";
+  // The property name for disabling caching of artifacts in GCS uploaded to GCS Bucket used by Dataproc.
+  // // It can be overridden by profile runtime arguments (system.profile.properties.enable.gcs.caching)
+  public static final String ENABLE_GCS_CACHING = "enable.gcs.caching";
```

Comment:
Sorry that I didn't pay attention to existing config naming convention. It seems like we are using camel case instead. Maybe better have it as `GCS_CACHE_ENABLED = "gcsCacheEnabled";`?

Reply:
ack